### PR TITLE
🔧 fix(spark stop-loss): Fix remaining issues with spark stop loss

### PR DESCRIPTION
**Pull request by BackToTheCode**
# [🔧 Fix spark stop-loss](https://app.shortcut.com/oazo-apps/story/11987/spark-stop-loss-finish-up)

## Changes 👷‍♀️
- Fix coverage issue for Spark Stop Loss
  
## How to test 🧪
- `yarn test`

### CODE REVIEW

File: features/automation/protection/stopLoss/state/stopLossTriggerData.ts [hunk 1]
```diff
@@ -139,10 +139,17 @@ export function prepareStopLossTriggerDataV2(
   debtTokenAddress: string,
   tokenAddress: string,
 ) {
+  let _maxCoverage = maxCoverage // maxCoverage, equals to 1500 USDC
+  if (
+    triggerType === TriggerType.SparkStopLossToDebtV2 ||
+    TriggerType.SparkStopLossToCollateralV2
```
Comment (piotrwitek) [inline features/automation/protection/stopLoss/state/stopLossTriggerData.ts:145]: @BackToTheCode Did you miss the comparison here for `TriggerType.SparkStopLossToCollateralV2`?

File: scripts/docker/compose.yml [hunk 2]
```diff
@@ -13,22 +13,22 @@ services:
...
+#      - ETHERSCAN_API_KEY=YAJI4NVD8QTQ9JVWG2NKN3FFUK6IZTMV5S
+#      - PRIV_KEY_MAINNET=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+#      - TENDERLY_USERNAME=georgi2
+#      - TENDERLY_PROJECT=mpoc
+#      - USE_DUMMY=${USE_DUMMY}
+#      - BLOCK_NUMBER=${BLOCK_NUMBER}
+#    ports:
+#      - '8545:8545'
+#    volumes:
+#      - '../.hardhat/cache:/usr/src/mpa/cache'
```
Comment (piotrwitek) [inline scripts/docker/compose.yml:32]: Is this intended? as it looks to me like out of the scope of the fix

**Review by piotrwitek — CHANGES_REQUESTED** [review]

File: scripts/docker/compose.yml (hunk 2 above)
Comment (BackToTheCode) [reply scripts/docker/compose.yml:32]: TY. Not intended at all

File: features/automation/protection/stopLoss/state/stopLossTriggerData.ts (hunk 1 above)
Comment (BackToTheCode) [reply features/automation/protection/stopLoss/state/stopLossTriggerData.ts:145]: I did. TY :)

**Review by piotrwitek — APPROVED** [review]

**Pull request merged**